data-prepper-test-common/src/test/java/org/opensearch/dataprepper/test/matcher/MapEqualsTest.java

```diff
@@ -18,28 +18,33 @@
 import static org.opensearch.dataprepper.test.matcher.MapEquals.isEqualWithoutTimestamp;
 
 class MapEqualsTest {
-    private static final Map<String, Object> expectedMap = new HashMap<>();
+    private static final Map<String, Object> expectedMap1 = new HashMap<>();
+    private static final Map<String, Object> expectedMap2 = new HashMap<>();
 
     @BeforeAll
     static void setup() {
-        expectedMap.put("List", Arrays.asList(1, 2, 3));
-        expectedMap.put("@timestamp", OffsetDateTime.now());
+        expectedMap1.put("List", Arrays.asList(1, 2, 3));
+        expectedMap1.put("@timestamp", OffsetDateTime.now());
```

Comment:
Last nitpick: you could make the DEFAULT_KEY package private and import it here. That way we don’t have to replace it in all these tests if it changes